src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -83,36 +87,36 @@ $clr-signpost-innerBlock: $clr-white !default;
             measurements and they appear to hold up well.
           */
 
-
+        // TODO rem-refactor
          $clr-signpost-border-size: 0.5rem;
          // TODO: should be able to theme the colors in these pointers...
          $clr-signpost-pointer-border: $clr-signpost-border-size solid $clr-signpost-outerBorder;
          $clr-signpost-pointer-invisible-border: $clr-signpost-border-size solid transparent;
          $clr-signpost-pointer-psuedo-border: $clr-signpost-border-size solid $clr-signpost-innerBlock;
- 
+
```

Comment:
Formatting changes. If needed to be done they should be in a separate PR.

Reply:
I removed some variables from here (I don't think its reflected in this commit) and left one line between the comment and the start of the `&.top-left` rule.